### ISSUE TITLE
i2b2 dataset results

### ISSUE THREAD

**Issue by andreasgrv** [dmis-lab/biobert#41]
Hi, thanks for making your code and pre-trained weights publicly available.

I have a question regarding the i2b2 dataset results in Table 4 of your paper.

In your paper, the paper you cite as SOTA for the i2b2 dataset ([Zhu 2018](https://arxiv.org/pdf/1810.10566.pdf)) seems to be predicting all entity types (problem, treatment and test). However, in your code: https://github.com/dmis-lab/biobert/blob/7a3c96e3fca2129fe74d78f9b707397fedd4cbd9/biocodes/ner_detokenize.py#L85 you seem to be ignoring entity types before evaluation (MISC) - so if I understand correctly the model isn't penalised if it gets the label type wrong? Also, are you using all 3 entity types for fine tuning on i2b2?

**Comment by jhyuklee** [comment]
Hi, @andreasgrv, we only trained and tested on 'problem' entity in i2b2 dataset as we wanted to see our model's performance on the disease entity type. We contacted the authors (Zhu et al., 2018) for their model's performance on the problem entity only, but we couldn't get the exact number. But, we also heard from the authors that their performance on the problem entity was very similar to that on all entity types. Thank you.
